addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/PiiScanRuleUnitTest.java

```diff
@@ -152,6 +154,53 @@ public void shouldRaiseAlertInPlausiblePeriodDelimitedContent() throws Exception
         assertThat(alertsRaised.size(), is(1));
     }
 
+    @Test
+    public void shouldNotRaiseAlertOnCssResponse() throws Exception {
```

Comment:
This actually testing the request. For the response it would have to be a message with Content-Type header containing `css`.